include/powercontrol.h

```diff
@@ -42,17 +49,17 @@ class
 	volatile unsigned long lastPowerOffResetTimestamp = 0;
 
 	// caching the PIN state to avoid unnecessary calls to the GPIO register
-	volatile int currentPowerPinMode = LOW;
+	volatile int currentPowerPinMode = LED_POWER_INVERT ? LOW : HIGH;
```

Comment:
Does this still need to be depended on LED_POWER_INVERT if now the methods are invert?